Source/UETorch/Private/TorchContext.h

```diff
@@ -24,8 +24,10 @@ class FTorchContext : public FLuaContext
 
   void Tick(float DeltaTime);
 	bool CallFunctionString(const FString& FunctionName, FString In, FString& Out);
+	bool CallFunctionArray(const FString& FunctionName, TArray<FString> In, FString& Out);
```

Comment:
Should be `const TArray<FString> &in`.

This is fine for now, but we really need to move to something more robust, e.g.

`bool CallFunction(const FString const TArray<FLuaObject> in, FLuaObject& Out);`

where FLuaObject is a discriminated union of string, int, Object* etc.

I suppose this will all become less important once we move all this blueprints stuff to Lua.